interfaces/machine_parameters/ring.py

```diff
@@ -29,6 +29,16 @@
 from .ring_section import RingSection
 
 
+# Default options of the Ring object
+_Ring_opt_dflt = {}
+_Ring_opt_dflt['t_start'] = 0
+_Ring_opt_dflt['t_stop'] = np.inf
+_Ring_opt_dflt['interp_time'] = 0
+_Ring_opt_dflt['interpolation'] = 'linear'
+_Ring_opt_dflt['store_turns'] = True
+_Ring_opt_dflt['eta_orders'] = 0
+
```

Comment:
Maybe worth adding a little function to print the defaults?
```suggestion
def RingKeywordDefaults():
    for k in _Ring_opt_dflt:
        print(f"For keyword {k}, the default value is: {_Ring_opt_dflt[k]}")
```
What do you think?

Reply:
It's a good idea. I think we could have a more general approach since many if not all the modules will have default option values. Maybe let's keep the utilities to display and modify default option values for a specific discussion?